### ISSUE TITLE
How could I get only the changed fields?

### ISSUE THREAD

**Issue by popkin** [snikch/jquery.dirtyforms#117]
I want this plugin to monitor if my form is dirty, but just submit the changed fields to the server.
How can I do this?

**Comment by NightOwl888** [comment]
Dirty Forms is only concerned with *if* the form is dirty, not *what* is dirty. While it does track what is changed by storing the original value in a JQuery data element called `df-orig` (and if the original value is empty, it stores `true` in a data element named `df-empty`), there is no public interface for getting this data back out. So, you would need to devise a way to do this manually (see the [isFieldDirty](https://github.com/snikch/jquery.dirtyforms/blob/master/jquery.dirtyforms.js#L435-L438) method for ideas).

In addition, you would need to [customize event binding](https://github.com/snikch/jquery.dirtyforms#example-usage) to remove the events that fire the dialog when navigating away from the form.

```
    // must be placed before first call to `.dirtyForms()`
    $(document).bind('bind.dirtyforms', function (ev, events) {
        events.bind = function (window, document, data) {
            // don't bind any events here, just return
            return;
        };
    });
```

The bottom line is, submitting only changed fields to the server is not what Dirty Forms was designed for - its only purpose is to remind the user to save the form before navigating away. You might be better off writing your own plugin for this, or finding one that more closely matches your use case.
